config/config.go

```diff
@@ -90,6 +90,7 @@ type Config struct {
 	RelayNode      string   `long:"relaynode" description:"The address of relay node that routes traffic between two peers over a qitmeer “relay” peer."`
 	Whitelist      string   `long:"whitelist" description:"Add an IP network or IP that will not be banned. (eg. 192.168.1.0/24 or ::1)"`
```

Comment:
- Can the `LANPeers` and `WhiteList` to be merged into single name `WhiteList` for a better naming and simplify the usage. let the `WhiteList` to support IP address and `PeerID` both. 
- BTW, I don't see the `Whitelist` is used throughout the PR, What happens if `Whitelist` and `LANPeer` both configured.
- I think its good idea to keep using single input `WhiteList` at the first place to minimize the future inconsistence.

Reply:
https://github.com/Qitmeer/qitmeer/pull/559/commits/ffabda6b9d0ec67de105b1fd9d87a71a0fdb5393